esi/tests/unit/lib/test_node.py

```diff
@@ -0,0 +1,235 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+
+from esi.lib import node
+from esi.tests.unit import base
+from esi.tests.unit import utils
+
+
+class TestList(base.TestCommand):
```

Comment:
You don't want to extend TestCommand, since these aren't the OpenStack commands that are being tested. Look at test_utils.py for the model you want to follow to test utility commands. You'll end up wanting a class for each function.